### DISCUSSION TITLE
Updated media editor was announced somewhere or me trippin?

### DISCUSSION THREAD

**Discussion by peterparker2005** [payloadcms/payload#9941] [General]
I feel like I saw something about editing Media in a better way in admin panel. This was a dream or what lol

**Comment by akhrarovsaid** [comment]
I'm not sure exactly what you're describing, but you can "edit" uploaded image media by clicking the edit icon button next to your upload. This will allow you to modify cropping, inspect sizes, and adjust focal points. Learn about this [here](https://payloadcms.com/docs/upload/overview).

You might also be referring to the uploads folder feature which is still a WIP but should be released open-source in the future.

**Reply by peterparker2005** [reply]
yes, folders! but where I could see this?

**Reply by akhrarovsaid** [reply]
@sznski You likely saw this in the [Payload website here](https://payloadcms.com/use-cases/digital-asset-management). As it's a WIP and not released yet (at least not for non-enterprise folks), there isn't really a good location for me to direct you to. As I understand it, the roadmap hasn't been updated yet, but the folders feature is still being actively worked on.
